views/html/rankings/misc.html

```diff
@@ -58,6 +62,9 @@
                 <td class=right>{{ comma .Bytes }}
                 <td class=right>{{ comma .Chars }}
             {{ end }}
+            {{ if $hasMedal }}
+                <td>{{ if eq .GolferRank 1 }}🥇{{ end }}
```

Comment:
Traditionally we do this by putting `class=me` on the `tr` so it colours green, rather than adding a new column.